### PR TITLE
Add variables for all languages

### LINKED ISSUE #19: List with all languages in server section.
Can be useful to get all languages to list them in a select.

### PR DESCRIPTION
**Pull request by dannywillems**
Fix #19

### CODE REVIEW

File: i18n_generate.mll [hunk 1]
```diff
@@ -81,6 +81,14 @@ and parse_string_2 buffer = parse
 
 {
 
+let print_list_of_langs fmt langs =
+  let rec aux = function
+  | [] -> "]"
+  | [head] -> head ^ "]"
```
Comment (sagotch) [inline i18n_generate.mll]: Note: this case is not really useful as trailing semi colon is okay in OCaml.

Comment (dannywillems) [inline i18n_generate.mll]: @sagotch Do you want I remove it?

**Comment by dannywillems** [comment]
I reopen after a new discussion with @sagotch :).

**Comment by sagotch** [comment]
One more thing. We would need it in a shared section...

**Comment by dannywillems** [comment]
@sagoth Done.

**Comment by dannywillems** [comment]
I prefer `all_languages` instead of `langs` for the variable name. I change it.

**Comment by sagotch** [comment]
In this case, `available_languages` would be even better...

**Comment by dannywillems** [comment]
@sagotch Done.

**Comment by sagotch** [comment]
I actually think that `langs` is better and also shorter to write, so I merged your pull request without the new name.

**Pull request closed without merging**